Added a gitignore and removed files that should be ignored

rpecka opened the pull request.

rpecka committed:
Added a gitignore and removed files that should be ignored

jfdoming reviewed (commented):
Can you add file types for exported projects as well (e.g. *.exe on Windows)? Otherwise LGTM.

rpecka committed:
Added an export directory where eported binaries can be placed

rpecka committed:
Added installation instructions to the readme

rpecka commented:
@jfdoming There are quite a few formats so I added an ignore for an export directory. Anything placed in there would be ignored. I also added installation instructions to the README so that people would create that dir

jfdoming reviewed (approved):
👍

rpecka committed:
Updated .gitignore comment
